include/ogdf/internal/decomposition/TricComp.h

```diff
@@ -49,14 +49,30 @@ namespace ogdf {
 class TricComp
 {
 public:
-	//! construction
+	/**
+	 * Divides G into triconnected components.
+	 * \param G graph
+	 */
 	TricComp(const Graph &G);
 
+	/**
+	 * Tests G for triconnectivity.
+	 * \param G graph
+	 * \param isTric true if G is triconnected, false otherwise.
+	 * \param s1 first vertex of separation pair if G is biconnected, cut vertex of G if G is not biconnected, nullptr if G is not connected.
+	 * \param s2 second vertex of separation pair if G is biconnected, nullptr otherwise.
+	 */
 	TricComp(const Graph &G, bool &isTric, node &s1, node &s2);
 
+	/**
+	 * Copy constructor.
```

Comment:
This might be a bit misleading as the copy constructor is [not available](http://en.cppreference.com/w/cpp/language/copy_constructor).